src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecInputModelTypeConverter.cs

```diff
@@ -135,58 +135,65 @@ private static InputModelType CreateInputModelTypeInstance(string? id, string? n
             return model;
         }
 
-        private static void CreateProperties(ref Utf8JsonReader reader, ICollection<InputModelProperty> properties, JsonSerializerOptions options, bool isMultipartType)
+        private static void CreateProperties(ref Utf8JsonReader reader, ICollection<InputModelProperty> properties, JsonSerializerOptions options, string modelName, bool isMultipartType)
```

Comment:
Is parameter `modelName` only used in a log? If so, shall we add this parameter?

Reply:
It's used to compose exception message to make things easier to debug.